src/System.IO.Compression/tests/ZipArchive/zip_InvalidParametersAndStrangeFiles.cs

```diff
@@ -185,12 +185,11 @@ public static void InvalidConstructors()
         }
 
         [Fact]
-        [ActiveIssue(1904, PlatformID.AnyUnix)]
         public static async Task StrangeFiles4()
         {
             ZipTest.IsZipSameAsDir(await StreamHelpers.CreateTempCopyStream(
                  ZipTest.strange("dataDescriptor.zip")), ZipTest.zfolder("normalWithoutBinary"), ZipArchiveMode.Update, true, false);
-	    }
+        }
```

Comment:
Do we have any tests that create appropriate archives on one OS and validate them on another?  If we don't, I expect this would involve creating a few additional .zip files and adding them to the test assets we pull down as part of test runs.